coscale-dev-env/deploy-agent.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
The script should be used on a resource group with VMs that you want to monitor, not the resource group used in the template.